### PR TITLE
docs: adds documentation for mentor routes

### PR DESCRIPTION
**Pull request by iamaamunir**
This PR...

## Changes

-

## Fixes
-

## Checklist

- [x] tested locally
- [x] added new dependencies
- [x] updated the docs
- [ ] added a test
- [x] linter passes
- [x] format check passes

Fixes #

### CODE REVIEW

**Comment by iamaamunir** [comment]
This pull request adds the documentation for the mentor routes. The documentation was done with Swagger UI

**Review by misspee007 — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 const { errorHandler } = require('./utils/errorHandler');
 require('express-async-errors');
 
+const swaggerJsDocs = YAML.load('./api-mentor.yaml');
+
 const indexRouter = require('./routes/index');
 
 const app = express();
 
+app.use('/api-docs-mentor', swaggerUI.serve, swaggerUI.setup(swaggerJsDocs));
```
Comment (tobisupreme) [inline app.js]: Please let's use `/api-docs` so that we can add other components as titles

File: api-mentor.yaml [hunk 2]
```diff
@@ -0,0 +1,314 @@
+openapi: 3.0.0
+info:
+  title: CoStudy Mentors Docs
```
Comment (tobisupreme) [inline api-mentor.yaml]: I think that we can change the title to 'CoStudy API Docs'

**Review by tobisupreme — APPROVED** [review]

**Comment by tobisupreme** [comment]
@all-contributors add @iamaamunir for documentation

**Pull request merged**